gradle.md

```diff
@@ -113,25 +143,30 @@ Gradle automatically tracks task dependencies, so if you modify a file in `/src/
       # The resulting files are placed under build/publications folder
       gw generatePom
 
-## Release artifacts
+## Release Artifacts
 
       # Builds ZIP and TGZ artifacts for the release
       gw :src:dist:assemble
 
 ## Signing
 
-It is implemented via [gradle signing plugin](https://docs.gradle.org/5.2.1/userguide/signing_plugin.html), so it is done automatically provided credentials are specified via [signatory credentials](https://docs.gradle.org/5.2.1/userguide/signing_plugin.html#sec:signatory_credentials)
+It is implemented via [gradle signing plugin](https://docs.gradle.org/5.2.1/userguide/signing_plugin.html), 
+so it is done automatically provided credentials are specified via
+[signatory credentials](https://docs.gradle.org/5.2.1/userguide/signing_plugin.html#sec:signatory_credentials)
 
-      # Signs all the artifacts of the current module (see results in build/**/*.asc
+      # Signs all the artifacts of the current module
+      # see results in build/**/*.asc
       gw sign
-> **Note:** signing is performed as a part of *release artifact build* so it will be performed with `gw :src:dist:assemble`
+> **Note:** signing is performed as a part of *release artifact build* so it will be 
+> performed with `gw :src:dist:assemble`
 
 ## Releasing
 
       # Builds the project, pushes artifacts to svn://.../dev, stages artifacts to Nexus staging repository
       gw prepareVote -Prc=1 -Pasf
 
-> **Note:** The above step uses [an asf-like release environment](https://github.com/vlsi/asflike-release-environment), so it does not alter public repositories
+> **Note:** The above step uses [an asf-like release environment](https://github.com/vlsi/asflike-release-environment),
+> so it does not alter public repositories
```

Comment:
Ah. That is a bug `-Pasf` means "publish to ASF servers"

Reply:
I'm not sure what you mean,  would you be able to merge the PR and then correct this section yourself? Also I noticed it mentions SVN, I presume this is no longer true?

Reply:
SVN is used for all Apache projects for distributing the releases via https://dist.apache.org/repos/dist